web-client/src/views/CaseListTable.tsx

```diff
@@ -130,6 +130,7 @@ export const CaseListTable = connect(
                   defaultActiveTab={openTab}
                 >
                   <Tab
+                    data-testid="open-cases-count"
```

Comment:
For the future, what do you (and the rest of the team) think of trying to name `data-testid` attributes semantically? I have trying to follow a pattern of `<element_type>-<feature>`. For example, here I might use `tab-open-cases`